modules/rostests/apitests/ntdll/RtlUnicodeToOemN.c

```diff
@@ -216,7 +216,7 @@ START_TEST(RtlUnicodeToOemN)
                                       Length,
```

Comment:
Shouldn't this need a (ULONG) cast as well? (since the parameter is ULONG but Length is SIZE_T)

Reply:
Moreover, isn't (silent) truncation bad?
And line 229 should use a `SIZE_T` too, then.

Reply:
It's a stupid test. There isn't anything "critical" about it. And please stop with those "but then you have to fix xyz, too"! No, I don't! I fixed some warnings. There are no more warnings now. So I don't have to "fix" anything. PR reviews are not meant to tell people what else they could fix or to tell them what flavor of ice cream you like, they are there to prevent introducing bugs! If you would focus on this, instead of other nonsense, it would help us all a lot.